spec/controllers/orders_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 describe  OrdersController  do
-  describe "index'" do  	
+  describe "Get index" do  	
```

Comment:
Use uppercase for HTTP methods names in specs, for example, there should be `GET`.